### PR TITLE
Setup fixes 2020

### PR DESCRIPTION
**Pull request by GalGend**
Hi,

I set up a new environment and updated the setup manual.
Also, fixed the migration scripts for several schemas

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,12 +52,6 @@ CREATE DATABASE meirim character set UTF8 collate utf8_bin;
 exit;
 ```
 
-Import the sql file
```
Comment (GalGend) [inline README.md]: No need to import, the migrations create the schemes

File: api/model/person.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 const Promise = require('bluebird');
-const Bcrypt = require('bcrypt');
```
Comment (GalGend) [inline api/model/person.js]: BcryptJS is lighter and in pure JS

Comment (dortheimer) [reply api/model/person.js]: Is it backward compatible?
Does it require a different node version?

Comment (florpor) [reply api/model/person.js]: I'm also not sure about using this bcyptjs - the [original](https://www.npmjs.com/package/bcryptjs) package was not updated for about 3 years now, and this [@jswebfans/bcryptjs](https://www.npmjs.com/package/@jswebfans/bcryptjs) package is some fork with this single release and without a codebase (all links still refer to the original package source).

And regarding @dortheimer 's comment - maybe we should decide on the project's node version and cement it (.nvmrc or w/e method) so we are all on the same page. Currently it seems the dependencies won't install on node version 12 onwards...

Comment (dortheimer) [reply api/model/person.js]: @GalGend, can you change this back?

**Comment by GalGend** [comment]
@dortheimer @florpor 
Changed back to Bcrypt

**Comment by florpor** [comment]
@GalGend is this ready for merge?

**Comment by GalGend** [comment]
@florpor It is

**Pull request merged**